msteams-platform/bots/streaming-ux.md

```diff
@@ -0,0 +1,586 @@
+---
+title: Stream bot messages
+description: Learn how to enhance the user experience in bots using streaming techniques.
+ms.date: 10/23/2024
+ms.topic: conceptual
+author: surbhigupta12
+ms.author: surbhigupta
+ms.localizationpriority: high
+---
+
+# Stream bot messages
+
+>[!NOTE]
+>
+> - Streaming bot messages is available only for one-on-one chats and in [public developer preview](../resources/dev-preview/developer-preview-intro.md).
+> - Streaming bot messages isn't available with Tools and the `o1` model.
+
+You can stream bot messages to deliver a bot's responses to the user as small updates while the complete response is being generated to enhance the user experience. Often, bots take a long time to generate responses without updating the user interface, leading to a less engaging experience.
```

Comment:
Discussion with Yunny/Yoga - do we need to note somewhere that this is only for specific tenants and in a personal 1:1 chat?

Reply:
@YunnyChung @yogapat Please help with this query.

Reply:
Updated as per Yunni's input.